src/web/components/SharingPermission/SharingPermissionsTable.tsx

```diff
@@ -3,41 +3,55 @@ import { CheckedState } from '@radix-ui/react-checkbox';
 import { useEffect, useMemo, useState } from 'react';
```

Comment:
I believe the plan is to also rename the 'Added By' column to `Source`, and then the rows will be something like 'DSP', 'Manually Added', 'DSP and Advertiser' and 'DSP and Manually Added'. Is the plan to do this in a future PR?

Reply:
I could do it in this PR